### PR TITLE
added truth tables for multiplexers

### PR DESCRIPTION
**Pull request by Dishebh**
#### Changes done:
- [x] Added truth tables for multiplexers

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

File: docs/MSI/mux.md [hunk 1]
```diff
@@ -37,10 +37,33 @@ E is called the strobe or enable input which is useful for the cascading. It is
 
 ### 4 : 1 multiplexer
 The 4 : 1 multiplexer has 4 inputs and 2 control signals.
+
+#### Truth Table
+
+|S1|S2|Output(Y)|
```
Comment (Shivansh2407) [inline docs/MSI/mux.md:43]: Keep this Y(Output) as in other tables to make it consistent.

**Comment by Dishebh** [comment]
@Shivansh2407 Sure :+1: Also, the simulations are not labelled. They need to be fixed too, to get in sync with their corresponding with truth tables. Will do the changes related to simulations in my next PR.

**Comment by Dishebh** [comment]
@Shivansh2407 please let me know if the changes are fine.

**Comment by Dishebh** [comment]
@abhinavsri360 like all truth tables together under one section?

**Comment by abhinavsri360** [comment]
@Dishebh I was thinking to have all Truth table under the section 'Multiplexers come in multiple variations' because as of now this looks ugly. Also, I was thinking that if what I said is implemented in that case it would mess up the TOC again. What do you think? Isn't that looking ugly?

**Comment by Dishebh** [comment]
@abhinavsri360 tbh, it looks odd to me as well. But having all truth tables under one section would be inconvenient as well (for the end user's). Let's see if we can come up with a better plan for a more efficient organization of items.

**Comment by abhinavsri360** [comment]
@Dishebh How about removing the Truth table from TOC 🤔

**Comment by Dishebh** [comment]
That could be done. Let's have reviews from others as well. Mentioning for review and opinions @Shivansh2407 @YashKumarVerma @schezfaz

**Comment by abhinavsri360** [comment]
Yeah, that will be a better option to go with.

**Comment by schezfaz** [comment]
Yeah I think we should exclude truth tables from TOC as we aren't mentioning block diagrams either,  so it only makes sense to exclue them. 

Also, could you align the truth table contents in a standardized way throughout the page? Some truth tables are left-aligned while some are center-aligned. I believe most truth tables across the book are center aligned.

For ref: 
image
and
image

**Comment by Dishebh** [comment]
@schezfaz updated the truth table. Thanks for the review :+1:

**Comment by Dishebh** [comment]
Tagging for review: @Shivansh2407 @YashKumarVerma

**Comment by Dishebh** [comment]
@Shivansh2407 removed truth tables from toc

**Pull request merged**